Gemfile

```diff
@@ -5,4 +5,5 @@ group :test do
   gem 'rspec', '~> 3.7.0'
   gem 'watir'
   gem 'ffi'
+  gem 'page-object'
```

Comment:
You should always add the version to the gem like you did with rspec. this way you can ensure everyone is using the versions.